.travis/osx..install.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-PACKAGES="cmake pkg-config fftw libogg libvorbis libsndfile libsamplerate jack sdl stk portaudio node fltk"
+PACKAGES="cmake pkgconfig fftw libogg libvorbis lame libsndfile libsamplerate jack sdl libgig libsoundio stk portaudio node fltk"
```

Comment:
@tresf That may make sense. However, I think leave `pkg-config` is the better.
It was `pkg-config` since #3476 by @zapashcanon,  See https://github.com/LMMS/lmms/commit/7844b3e289e3627bf89d5c3048ac2ab03b33fff5#diff-199dc21cb5e108771eecb4815a82b450R5.